lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
I don't know what the spec says but this behavior looks like a V8 bug. I was expecting two possibilities:
* each iteration creates a new scope, so a new const value and it works
* there is only one scope for the loop and it throws after the first iteration because we can't reassign x
Here we have a mix of both :confused: